x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -119,6 +132,92 @@ export function createExecutionEnqueuerFunction({
   };
 }
 
+export function createBulkExecutionEnqueuerFunction({
+  taskManager,
+  actionTypeRegistry,
+  isESOCanEncrypt,
+  preconfiguredActions,
+}: CreateExecuteFunctionOptions): BulkExecutionEnqueuer<void> {
+  return async function execute(
+    unsecuredSavedObjectsClient: SavedObjectsClientContract,
+    options: ExecuteOptions[]
```

Comment:
nit: would be nice to have a more descriptive name like `actionsToExecute` or `actionsToEnqueue`

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137781/commits/b15ed37052ae2e0e865df502af4255665c206408)